src/pages/login/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+const Login = () => {
+  const navigate = useNavigate();
+  return (
+    <S.LoginPageWrapper>
+      <S.LogoBox>로고</S.LogoBox>
+      <S.SocialLoginButton
+        onClick={() => {
+          navigate('/');
+        }}
```

Comment:
```suggestion
        handleLoginButtonClick = () => { navigate('/')}
        ...
        onClick={handleLoginButtonClick}
```
로직이 JSX에 들어가는 걸 지양하려면 위처럼 작성하는 것도 좋아보입니다